Bento/Views/BentoReusableView.swift

```diff
@@ -1,10 +1,20 @@
+/// Existential type for reusable views (cell, header, footer) in `UITableView`, `UICollectionView`, etc.
```

Comment:
A protocol is not an existential.

Reply:
Can you provide more information about why you think so?

P.S. `protocol  BentoReusableView` is currently _used_ an existential only.

Reply:
It is purely terminology. How a protocol is used, generics or existentials, does not change what the protocol is, which is a static-time contract with semantics.

Reply:
0aeae60 Updated to not use jargon.